src/Generators/Microsoft.Gen.MetricsReports/Directory.Build.props

```diff
@@ -4,6 +4,8 @@
   the project itself. -->
   <PropertyGroup>
     <Stage>dev</Stage>
+    <StageDevDiagnosticId>EXTEXP0018</StageDevDiagnosticId>
```

Comment:
I'm actually curious now, which Package is this in? I don't think this generator has it's own package, right?